Source/AliveLibAE/Abe.hpp

```diff
@@ -300,58 +300,58 @@ struct Abe_SaveState
     DWORD slapable_or_pickup_id;
     DWORD wheel_id;
     DWORD invisible_timer;
-    WORD wordA0;
-    WORD wordA2;
-    char byteA4;
+    WORD field_A0_unused;
+    WORD field_A2_invisibility_id;
+    char field_A4_cam_idx;
     char field_A5_padding;
     __int16 field_A6_padding;
     DWORD hand_stone_type;
     WORD fmv_id;
     WORD cam_id_1;
     WORD cam_id_2;
     WORD cam_id_3;
-    WORD wordB4;
-    WORD wordB6;
-    WORD wordB8;
-    WORD wordBA;
-    WORD wordBC;
-    WORD wordBE;
+    WORD field_B4_unused;
+    WORD field_B6_unused;
+    WORD field_B8_unused;
+    WORD field_BA_unused;
+    WORD field_BC_unused;
+    WORD field_BE_unused;
     WORD bHaveEvilFart;
     LevelIds to_level;
     WORD to_path;
     WORD to_camera;
     WORD door_id;
     char field_ca_throw_direction;
-    char field_CB;
-    WORD wordCC;
+    char field_CB_padding;
+    WORD field_CC_portal_sub_state;
     __int16 field_CE_padding;
     DWORD bird_portal_id;
     enum Flags_D4
     {
-        eD4_Bit1 = 0x1,
-        eD4_Bit2 = 0x2,
-        eD4_Bit3 = 0x4,
-        eD4_Bit4 = 0x8,
-        eD4_Bit5 = 0x10,
-        eD4_Bit6 = 0x20,
-        eD4_Bit7 = 0x40,
-        eD4_Bit8 = 0x80,
-        eD4_Bit9 = 0x100,
-        eD4_Bit10 = 0x200,
-        eD4_Bit11 = 0x400,
-        eD4_Bit12 = 0x800,
+        eD4_Bit1_lift_point_dead_while_using_lift = 0x1,
+        eD4_Bit2_return_to_previous_motion = 0x2,
+        eD4_Bit3_fall_to_well = 0x4,
+        eD4_Bit4_unused = 0x8,
+        eD4_Bit5_prevent_chanting = 0x10,
+        eD4_Bit6_land_softly = 0x20,
+        eD4_Bit7_unused = 0x40,
+        eD4_Bit8_laugh_at_chant_end = 0x80,
+        eD4_Bit9_unused = 0x100,
+        eD4_Bit10_play_ledge_grab_sounds = 0x200,
+        eD4_Bit11_unused = 0x400,
+        eD4_Bit12_have_healing = 0x800,
         eD4_eBit13 = 0x1000,
```

Comment:
bit 13 not named?

Reply:
It connects to `Flags_114::e114_Bit10` in the `BaseAliveGameObject` class, so I'll tackle that one later.